src/FSharp.Editing/FSharp.Editing.fsproj

```diff
@@ -39,7 +39,7 @@
     <DocumentationFile>bin\Release\FSharp.Editing.XML</DocumentationFile>
     <OtherFlags>--warnon:1182</OtherFlags>
     <NoWarn>52</NoWarn>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I can change it back later, but until the `SmartTag` deprecated warning gets removed it stops me from being able to build everything. With all of the namespace changes and reorganization I've been doing this setting is incredibly annoying.